### DISCUSSION TITLE
Headless with AMD?

### DISCUSSION THREAD

**Discussion by Mannshoch** [LizardByte/community#490] [Sunshine]
Nearly every guide I find use Nvidia GPU.
Goal would be a Gameserver in the cellar with AMD-GPU without screen connected and may an instant created virtual screen in the size of the connecting device. eg TV or android device.
Is there someone with a working setup that way?

**Comment by minosimo** [comment]
There's a reddit thread: https://www.reddit.com/r/linux_gaming/comments/199ylqz/streaming_with_sunshine_from_virtual_screens/

**Reply by Mannshoch** [reply]
Is there a way, may Sunshine could add support for an automatism for that?

**Reply by ReenigneArcher** [reply]
Unlikely... too many different scenarios on Linux.

[a comment by an automated account is not shown]
